src/main/java/teamrtg/rtg/core/event/EventManagerRTG.java

```diff
@@ -33,238 +35,349 @@
 import teamrtg.rtg.core.world.gen.structure.MapGenVillageRTG;
 import teamrtg.rtg.core.world.gen.structure.StructureOceanMonumentRTG;
 
-public class EventManagerRTG {
+public class EventManagerRTG
+{
 
+    private final LoadChunkRTG LOAD_CHUNK_RTG = new LoadChunkRTG();
+    private final GenerateMinableRTG GENERATE_MINABLE_RTG = new GenerateMinableRTG();
+    private final InitBiomeGensRTG INIT_BIOME_GENS_RTG = new InitBiomeGensRTG();
+    private final InitMapGenRTG INIT_MAP_GEN_RTG = new InitMapGenRTG();
+    private final SaplingGrowTreeRTG SAPLING_GROW_TREE_RTG = new SaplingGrowTreeRTG();
+    
+    public final WorldEventRTG WORLD_EVENT_RTG = new WorldEventRTG();
+    public final RTGEventRegister RTG_EVENT_REGISTER = new RTGEventRegister();
+
+    private WeakHashMap<Integer, Acceptor<ChunkEvent.Load>> chunkLoadEvents = new WeakHashMap<>();
+    private boolean registered = false;
+    private long worldSeed;
+    private final String EVENT_SYSTEM = "RTG Event System: ";
```

Comment:
If we decide to change the prefix, then we only need to change it one place. With all the crazy decoration, surfacing and terrain generation we're doing, I think we can afford to create an extra string ;)